### ISSUE TITLE
subfolders missing from Crashplan

### ISSUE THREAD

**Issue by kiddsupreme** [jlesage/docker-crashplan-pro#179]
Not sure if this has been answered or not, but thought I try asking here. I am currently running a Fedora 30 system. When I first installed the docker container, I had it setup as such:

`docker run -d \
    --name=crashplan-pro \
	--restart=always \
    -p 5800:5800 \
    -v /opt/crashplan-pro:/config:rw \
    -v /:/storage:ro \
	-e GROUP_ID=1000 -e USER_ID=1000  \
	-e TZ=America/Los_Angeles \
	-e KEEP_APP_RUNNING=1 \
	-e SECURE_CONNECTION=1 \
	-e CRASHPLAN_SRV_MAX_MEM=2048M \
	jlesage/crashplan-pro`

Initially, this worked well; I was able to select the folders that I wanted to backup, and after about 8 days, everything was backed up -- or so I thought. Recently, I added more files to my system, and assumed that since they were

1. Under the same subfolders
2. Had the same permissions as the other files

Everything would be backed up. However, I noticed everyday that it would "say" that things had been backed up, but the amount would ALWAYS be 0 bytes. So I decided to investigate further. I clicked on "Manage Files" and browsed to one of the folders that should be getting backed up. When I click on that folder, everything under it was missing -- showing as empty. I double checked to make sure the files were indeed still there. I then issued the following:

docker exec -it crashplan-pro sh

Then made my way to /storage. From the container itself, I could ALSO see that all files that should be seen WERE being seen -- nothing was missing. But in the CrashPlan application, the folders had that little icon (a circle with a squiggly line denoting that the file was missing).

At this point, I am kind of at a loss for what the issue is. I went so far as to delete the container, delete the configuration folder, and start from the scratch. Yet, the CrashPlan application STILL shows these folders as missing. Any idea what may be going on? Thanks in advance.

**Comment by jlesage** [comment]
This looks like a permission issue.  Are you sure that the problematic folders/files have the proper permissions?
Also, are you required to map the entire host's root folder?

**Comment by kiddsupreme** [comment]
> This looks like a permission issue. Are you sure that the problematic folders/files have the proper permissions?
> Also, are you required to map the entire host's root folder?

Permissions are correct. However, I did notice something strange after posting my question. I went back into the GUI, and notice /storage folder. When I click into it, then I saw all the sub folders displaying correctly. Here’s the thing though: that /storage folder didn’t show up in my initial backup. Regardless, I went ahead and selected the items that should be backed up, and thankfully CrashPlan was smart enough to not make me re-upload the already backed up items. 

Not really sure what happened, just glad it’s working. Thank you again for the support; I definitely appreciate it.
